.github/workflows/build-test-deploy.yml

```diff
@@ -566,7 +566,11 @@ jobs:
   check-simnode:
     name: "Simnode"
     needs: package-simnode-tests
-    runs-on: ubuntu-latest
+    runs-on:
```

Comment:
speed for running on sre vs gh node for almost exactly same. and that fixed ownership issue. you may have returned back owership. but other stuff is useful

Reply:
It's currently broken because GH runners have size limitation that our backup exceed